crates/bevy_reflect/bevy_reflect_derive/src/derive_data.rs

```diff
@@ -174,7 +185,11 @@ impl<'a> ReflectDerive<'a> {
                 }
                 #[cfg(feature = "documentation")]
                 Meta::NameValue(pair) if pair.path.is_ident("doc") => {
-                    if let syn::Lit::Str(lit) = pair.lit {
+                    if let syn::Expr::Lit(syn::ExprLit {
+                        lit: syn::Lit::Str(lit),
```

Comment:
nit: Would it be possible to reduce the line noise by selectively importing `Expr`, `ExprLit` or `Lit`?

Reply:
this part is behind a feature, so it would mean putting the import also behind a feature, I find it less noisy that way